deployments/README.md

```diff
@@ -137,3 +138,10 @@ This is because this `deployments.json` file is used by bots in [`@across-protoc
 | ---------------- | -------------------------------------------------------------------------------------------------------------------------------- |
 | Ink_SpokePool    | [0xeF684C38F94F48775959ECf2012D7E864ffb9dd4](https://explorer.inkonchain.com/address/0xeF684C38F94F48775959ECf2012D7E864ffb9dd4) |
 | MulticallHandler | [0x924a9f036260DdD5808007E1AA95f08eD08aA569](https://explorer.inkonchain.com/address/0x924a9f036260DdD5808007E1AA95f08eD08aA569) |
+
+## Cher mainnet (1868)
+
+| Contract Name    | Address                                                                                                               |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------- |
+| Cher_SpokePool   | [0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96](https://etherscan.io/address/0x3baD7AD0728f9917d1Bf08af5782dCbD516cDd96) |
```

Comment:
To be clear, we are doing this since we don't want to expose the block explorer?

Reply:
yes. Will update the urls today/tomorrow